src/django_components/context.py

```diff
@@ -19,11 +20,17 @@
 _FILLED_SLOTS_CONTENT_CONTEXT_KEY = "_DJANGO_COMPONENTS_FILLED_SLOTS"
 _OUTER_ROOT_CTX_CONTEXT_KEY = "_DJANGO_COMPONENTS_OUTER_ROOT_CTX"
 _SLOT_COMPONENT_ASSOC_KEY = "_DJANGO_COMPONENTS_SLOT_COMP_ASSOC"
+_PARENT_COMP_KEY = "_DJANGO_COMPONENTS_PARENT_COMP"
+_CURRENT_COMP_KEY = "_DJANGO_COMPONENTS_CURRENT_COMP"
```

Comment:
On the Context object we now store the ID of the current component node that we're rendering, as well as it's parent.

The parent info is used in this MR as an indicator of whether the rendered component node is a top-level node, or it is inside another component.

If I remember right, the info on current component ID is / will be used to fix https://github.com/EmilStenstrom/django-components/issues/453